src/main/java/uk/ac/ebi/eva/pipeline/model/PopulationStatistics.java

```diff
@@ -15,17 +15,23 @@
  */
 package uk.ac.ebi.eva.pipeline.model;
 
+import org.springframework.context.annotation.Profile;
 import org.springframework.data.annotation.PersistenceConstructor;
 import org.springframework.data.mongodb.core.index.CompoundIndex;
 import org.springframework.data.mongodb.core.index.CompoundIndexes;
 import org.springframework.data.mongodb.core.mapping.Document;
 import org.springframework.data.mongodb.core.mapping.Field;
 
+import uk.ac.ebi.eva.pipeline.Application;
+
 import java.util.Map;
 
 /**
- * setters have package visibility, the user should use the constructor.
+ * Statistics related to a set of samples for a given variant.
+ * <p>
+ * Setters have package visibility, the user should use the constructor to instantiate valid objects.
  */
+@Profile(Application.MONGO_EXPERIMENTAL_PROFILE)
```

Comment:
Shouldn't the profile be applied to the `StatisticsMongoWriter` class too (or instead)? What would happen if the writer is injected and the model is not?